tests/unit/plugins/service/test_event_consumer.py

```diff
@@ -351,6 +361,40 @@ def mock_create_gevent_msg(consumer, mocker, monkeypatch):
     assert 3 == len(caplog.records)
 
 
+@pytest.mark.asyncio
+async def test_handle_pubsub_msg_old(mocker, monkeypatch, consumer,
+                                     raw_msg_data, creation_audit_log_data,
+                                     caplog, pubsub_msg, mock_get_and_validate,
+                                     mock_create_gevent_msg):
+    """Too-old message."""
+    mocker.patch(DATETIME_PATCH, conftest.MockDatetime)
+    pubsub_msg.publish_time -= datetime.timedelta(
+        seconds=(consumer._max_msg_age + 1))
+    context = {
+        'msg_id': pubsub_msg.message_id,
+        'msg_age': consumer._max_msg_age + 1
+    }
+
+    event_msg = event_consumer.GEventMessage(
+        pubsub_msg, creation_audit_log_data, phase='consume')
+    mock_create_gevent_msg.return_value = event_msg
+
+    mock_run_coro_threadsafe = mocker.Mock()
+    patch = ('gordon_gcp.plugins.service.event_consumer.asyncio.'
+             'run_coroutine_threadsafe')
+    monkeypatch.setattr(patch, mock_run_coro_threadsafe)
```

Comment:
We could skip this if we just verify there was nothing put on the success_channel instead.

Reply:
ok

Reply:
(also copypasta)